src/storages/inRedis/TelemetryCacheInRedis.ts

```diff
@@ -0,0 +1,29 @@
+import { ILogger } from '../../logger/types';
+import { Method } from '../../sync/submitters/types';
+import { KeyBuilderSS } from '../KeyBuilderSS';
+import { TelemetryCacheAsync } from '../types';
+import { findLatencyIndex } from '../findLatencyIndex';
+import { Redis } from 'ioredis';
+
+export class TelemetryCacheInRedis implements TelemetryCacheAsync {
+
+  /**
+   * Create a Telemetry cache that uses a storage wrapper.
```

Comment:
Is this comment accurate?
```suggestion
   * Create a Telemetry cache that uses Redis.
```

Reply:
you are right Gaston. Updating the comment. Thanks